apps/distgo/pkg/script/script.go

```diff
@@ -92,3 +94,20 @@ func WriteAndExecute(buildSpec params.ProductBuildSpec, script string, stdOut, s
 	}
 	return nil
 }
+
+func GetBuildArgs(buildSpec params.ProductBuildSpec, script string) ([]string, error) {
+	stdoutBuf := bytes.Buffer{}
+	stderrBuf := bytes.Buffer{}
+	combinedBuf := bytes.Buffer{}
+	stdoutMW := io.MultiWriter(&stdoutBuf, &combinedBuf)
+	stderrMW := io.MultiWriter(&stderrBuf, &combinedBuf)
```

Comment:
The `combinedBuf` is used in the returned error. There is a check on the `stderrBuf` to check if its empty. Otherwise, error is returned. This code is already existing code. I just moved it to a separate file.